src/scripts/site.js

```diff
@@ -709,6 +709,10 @@ var AppInfo = {};
             onError: onRequireJsError
         });
 
+        require(["fetch"], function (fetch) {
+            console.debug("fetch is here", fetch);
```

Comment:
Why the debug log?

Reply:
This is from my patch. I tried to check module status:
* On PC and webOS 4, `fetch` is object
* On webOS 3, `fetch` is `undefined`, but it is works

Reply:
removed